test/specs/zoom.spec.js

```diff
@@ -396,6 +396,79 @@ describe('zoom', function() {
     }
   });
 
+  describe('drag with zoom.drag.modifierKey', function() {
+    for (const key of ['ctrl', 'alt', 'shift', 'meta']) {
+      for (const pressed of [true, false]) {
+        let chart, scaleX, scaleY;
+        it(`should ${pressed ? 'not ' : ''}change ${pressed ? 'without' : 'with'} key ${key}`, async function() {
```

Comment:
The 'not'/'' and 'without'/'with' are reversed

Reply:
Whoops, good catch, thanks!